deployment/roles/reverse/templates/nginx/conf.d/vhosts.conf.j2

```diff
@@ -2,11 +2,15 @@ include /etc/nginx/conf.d/upstream;
 
 server {
 
-  listen 443 ssl;
+  listen {{ ip_service }}:443 ssl;
   include {{ nginx_ssl_conf }}/ssl.conf;
+  include /etc/nginx/conf.d/proxy_params;
```

Comment:
Attention à ne pas forcément le commoniser.

Ce n'étais pas présent pour le bloc cas :

```
  location = /cas {
    return 302 $scheme://$host/cas/login;
  }
```

En particulier, l'ajout & suppression des headers type `Authorization` & `X-Forwarded-For` n'affectent-ils pas l'authentification sur cas ou les logs?